### DISCUSSION TITLE
Creating and Update Superset Users in Bulk through API, apache superset version 1.1.0

### DISCUSSION THREAD

**Discussion by ghost** [apache/superset#18300] [Q&A / Help]
### Expected results
In previous version of superset I was able to create and update bulk users through API's, but with superset version 1.1.0 I am not able to do the same.
Below is the link about what I am talking about.
https://github.com/apache/superset/issues/3083 

what you expected to happen.
I want to create superset users in bulk.

### Actual results
I am unable to create users in bulk.

### Environment

- superset version: 1.1.0
- python version: 3.7

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by zhaoyongjie** [comment]
Would you like to use the command line to create users? for instance
```
 superset fab create-user
```

**Comment by ghost** [comment]
@zhaoyongjie can you please let me know how to create user with multiple roles using the below command.

superset fab create-user

**Comment by zhaoyongjie** [comment]
@srikark16 

sorry for that the CLI only support single role to add.

```
$ superset fab create-user --role Alpha --username public_alpha --firstname dummy --lastname dummy --email dummy@dummy.com --password dummy
User public_alpha created.
```

FAB this part of the code, maybe it can be modified. feel free to contribute to FAB.

https://github.com/dpgaspar/Flask-AppBuilder/blob/b9b3259a9d47a14430144c8362215c2143f06aa4/flask_appbuilder/cli.py#L102-L108

**Comment by ghost** [comment]
@zhaoyongjie

Thank you, for the confirmation.
